### ISSUE TITLE
sapp-windows: glGetString missing

### ISSUE THREAD

**Issue by panpang** [not-fl3/miniquad#178]
I have a hybrid gpu laptop, and I wanted to know gl_vendor, and gl_version used, but even though glGetStringi is there glGetString is missing for sapp_windows. (I think it is there for some of the other platforms)
I tried glGetStringi with GL_VENDOR, GL_VERSION but it is getting Error 0x500/1280 GL_INVALID_ENUM. which is normal I guess.

So I added it with 
---
$ git diff
diff --git a/native/sapp-windows/src/gl.rs b/native/sapp-windows/src/gl.rs
index 1febfa6..8c5f78d 100644
--- a/native/sapp-windows/src/gl.rs
+++ b/native/sapp-windows/src/gl.rs
@@ -331,6 +331,7 @@ macro_rules! gl_loader {

 gl_loader!(
     fn glGetStringi(name: GLenum, index: GLuint) -> *const GLubyte,
+    fn glGetString(name: GLenum) -> *const GLubyte,
     fn glFramebufferTextureLayer(
         target: GLenum,
         attachment: GLenum,
---

and used it with this code 
---
    let gl_vendor: GLenum = 0x1F00;
    let gl_version: GLenum = 0x1F02;
    unsafe {
        let vendor_str = std::ffi::CStr::from_ptr(glGetString(gl_vendor) as *const i8)
            .to_str()
            .expect("can not convert gl_vendor c-string to utf-8");
        let version_str = std::ffi::CStr::from_ptr(glGetString(gl_version) as *const i8)
            .to_str()
            .expect("can not convert gl_version c-string to utf-8");
        println!("GL Vendor: {}", vendor_str);
        println!("GL Version: {}", version_str);            
    }
---
(also could not find enum definitions for GL_VENDOR, GL_VERSION)
It is working for me with a local modified copy of miniquad,
but I wanted to inform about this, and maybe get this added.

Can glGetString be added to sapp_windows, or am I doing something wrong to get this info?
or if there a reason that it can not be added, please ignore.
If this kind of addition is outside the scope of miniquad, also please ignore/close issue.

What is the official way to propose a change like this? Should I use the fork / merge request option?

Thanks for a fantastic and very fast compiling framework supporting so many platforms.

**Comment by not-fl3** [comment]
yes, in sapp-windows there are only functions used by miniquad, so a lot of things are missing

feel free to make a PR :)
